runtime/colors/vim.lua

```diff
@@ -0,0 +1,290 @@
+-- Name:         vim
+-- Description:  Vim's default color scheme
+-- Author:       Bram Moolenaar <Bram@vim.org>
+-- Maintainer:   The Vim Project <https://github.com/vim/vim>
+-- Website:      https://github.com/vim/vim
+-- License:      Same as Vim
+-- Last Updated: 2023 Aug 10
+
+-- This is Vim's default color scheme. It doesn't define the Normal
+-- highlighting, it uses whatever the colors used to be.
+
+vim.cmd('hi clear')
+vim.g.colors_name = 'vim'
+
+local set_hl, cmd = vim.api.nvim_set_hl, vim.cmd
```

Comment:
I don't think that's necessary; you only use `set_hl` once and `vim.cmd` a handful of times.

Reply:
For `vim.cmd()` - sure. It's just makes more aligned with `hi()` calls.
But `set_hl` is called on every `hi()` call. So unless there is some dark magic JIT involved, I think using local `set_hl` should be faster (by avoiding lookups of `vim.api`).

Reply:
Benchmarked similar to [this comment](https://github.com/neovim/neovim/pull/26658#issuecomment-1863260486).

Results were 557 ms with using `local set_hl` and 560 ms with plain `vim.api.nvim_set_hl` call inside `hi` call. So yeah, number of calls doesn't seem significant enough to justify using `local set_hl`.